src/guide/jbs-jdk-bug-system.md

```diff
@@ -72,7 +72,51 @@ A few things to keep in mind when filing an issue:
+
+Another aspect of an issue is when, the feature it is a part of, was added or removed from the JDK, which in either case limits the range of releases the issue impacts.  Knowing that a feature was removed before the oldest currently supported release means it can be resolved as will-not-fix.,
+
+##### Setting the Affects Versions field
+Set the [Affects Versions]{.jbs-field} field to the lowest release value that the bug is known to be reproducible on.
+
+* The [Affects Version/s]{.jbs-field} isn't meant to be an exhaustive list of releases the issue is relevant to - it should initially be set to the release the issue was reproduced or identified on, and by implication it will be relevant on all releases past that point (see the [Usage of (Rel)[-na]{.jbs-label} Label](#usage-of-rel-na-label)). If it's later found to be applicable to an earlier release family then adding that earlier release is encouraged if the issue needs to be fixed in that release.
+* Do not add additional release values to [Affects Version/s]{.jbs-field} for the same release family. E.g. If there is the value [11.0.2]{.jbs-value}, don't add [11.0.5]{.jbs-value}, [11.0.7]{.jbs-value} etc. Adding an additional value for a separate release family where it's still reproducible, e.g. JDK [21]{.jbs-value}, is encouraged, especially if there is not currently a feature release value set, or, it has been a few releases since it was last reproduced/reviewed. E.g. [Affects Version/s]{.jbs-field} is JDK [8]{.jbs-value} but it is still relevant to the latest mainline release.
+
+##### Usage of (Rel)-na Label
+
+Labels of the form [12-na]{.jbs-label} or [18-na]{.jbs-label} should be used when a bug is not applicable to a more recent release family.
+
+[Affects Version]{.jbs-field}: `7u111, 8u131`
+
+add the label [9-na]{.jbs-label} if the issue is not relevant to JDK 9 and above.
+Reasons why this would be the case include the fact that the source has been removed from a later release or rewritten such that the issue is no longer relevant.
+
+Do not:
+
+- use the label to indicate that a bug is not relevant to an earlier release, for example<br>[Affects Version]{.jbs-field}: `11.0.20, 17`<br> the label [8-na]{.jbs-label} would not be needed - as it doesn't have a JDK 8 release, or earlier, value in the [Affects Versions]{.jbs-field}, it is not relevant to JDK 8
+- add multiple -na labels: you only need one, for example `9-na, 10-na` is not need as [9-na]{.jbs-label} implies any following releases
+
+##### Usage of (Rel)-wnf Label
+
+labels of the form [7-wnf]{.jbs-label} or [9-wnf]{.jbs-label} should be used to indicate that a bug is not going to be fixed in a release that it is applicable to, both explicitly ([11-wnf]{.jbs-label} states it won't be fixed in JDK 11) or implicitly ([11-wnf]{.jbs-label} indicates it won't be fixed in JDK 8).
```

Comment:
```suggestion
Labels of the form [Rel-wnf]{.jbs-label} (e.g. [9-wnf]{.jbs-label}) should be used to indicate that a bug is not going to be fixed in a release that it is applicable to, or any earlier releases. For example, [11-wnf]{.jbs-label} states it won't be fixed in JDK 11 and implicitly indicates it won't be fixed in JDK 8 either).
```

Reply:
I suggest replacing `9-wnf` with `11-wnf` — the entire item would look consistent.

Reply:
done